var/spack/repos/builtin/packages/r-deriv/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RDeriv(RPackage):
+    """Deriv: Symbolic Differentiation
+
+    R-based solution for symbolic differentiation. It admits user-defined
+    function as well as function substitution in arguments of functions to be
+    differentiated. Some symbolic simplification is part of the work."""
+
+    homepage = "https://cloud.r-project.org/package=Deriv"
+    url      = "https://cloud.r-project.org/src/contrib/Deriv_4.1.2.tar.gz"
```

Comment:
This is an invalid URL but wouldn't be if you replace `2` with `3` since v4.1.3 is available at that location.

Reply:
That is what I was referring to in https://github.com/spack/spack/pull/22781#pullrequestreview-628215936. The 4.1.2 version would still work though as CRAN packages will use `list_url`.